nntrainer/src/conv2d_layer.cpp

```diff
@@ -53,10 +53,10 @@ int Conv2DLayer::initialize(bool last) {
     } else {
       bias.setZero();
     }
-    Tensor delK(gradDim);
+    Tensor delK(dim);
```

Comment:
let's fix the value of `gradDim` to be used than setting value here manually with `dim`. Or remove `gradDim` variable.

Reply:
yes. better to remove. Thanks.